pages/profile/[username].tsx

```diff
@@ -0,0 +1,138 @@
+import { BaseContainer } from "../../component/Page/BaseContainer";
+import { Helmet } from "react-helmet";
+import React, { useEffect, useState } from "react";
+import LayoutContainer from "../../component/LayoutContainer";
+import Image from "next/image";
+import { Header } from "../../component/Header";
+import Toggle from 'react-toggle'
+import { useGetProfile } from "../../services/profile/profileHooks";
+import { useRouter } from "next/router";
+import { ITokenEnum } from "../../shared/enum";
+import { GetServerSidePropsContext } from "next";
+import { RotatingTriangles } from 'react-loader-spinner'
+import { toast } from "react-toastify";
```

Comment:
please change relative import to alias import
e.g 
`import { BannerPage } from "component/Page/BannerPage"`